hudi-utilities/src/main/java/org/apache/hudi/utilities/streamer/HoodieStreamer.java

```diff
@@ -424,6 +439,11 @@ public static class Config implements Serializable {
     @Parameter(names = {"--config-hot-update-strategy-class"}, description = "Configuration hot update in continuous mode")
     public String configHotUpdateStrategyClass = "";
 
+    @Parameter(names = {"--ignore-checkpoint"}, description = "Set this config with a unique value, recommend using a timestamp value or UUID."
+        + " Setting this config indicates that the subsequent sync should ignore the last committed checkpoint for the source. The config value is stored"
+        + " in the commit history, so setting the config with same values would not have any affect.")
+    public String ignoreCheckpoint = null;
```

Comment:
can you add info on why would someone use this instead of --checkpoint. 
for eg, lets illustrate the earliest auto reset usecase

Reply:
addressed